src/redux/api.js

```diff
@@ -0,0 +1,130 @@
+import axios from 'axios';
+
+const baseURL = 'http://localhost:3000/api';
+
+const API = {
+  login: (user, success) => {
+    axios.post(`${baseURL}/users/sign_in`,
+      {
+        user,
+      })
+      .then((response) => {
+        success(response);
+      })
+      .catch((error) => {
+        success(error.message);
+      });
+  },
+  register: (user, success) => {
+    axios.post(`${baseURL}/users`,
+      {
+        user,
+      })
+      .then((response) => {
+        success(response);
+      })
+      .catch((error) => {
+        success(error.message);
+      });
+  },
+  fetchMotors: (success) => {
+    axios.get(`${baseURL}/motocycles`)
+      .then((response) => {
+        success(response);
+      })
+      .catch((error) => {
+        success(error.message);
+      });
+  },
+  fetchSingleMotor: (id, success) => {
+    axios.get(`${baseURL}/motocycles/${id}`)
+      .then((response) => {
+        success(response);
+      })
+      .catch((error) => {
+        success(error.message);
+      });
+  },
+  addMotor: (motor, success) => {
+    axios.post(`${baseURL}/motocycles`, {
+      motors: motor,
+    })
```

Comment:
Again the spelling of `motorcycles` is incorrectly spelled as `motocycles`

Reply:
Thank you for the feedback; I will implement it :+1: